identity-wallet/src/state/user_prompt.rs

```diff
@@ -33,9 +33,9 @@ pub enum CurrentUserPrompt {
         issuer_name: String,
         #[ts(optional)]
         logo_uri: Option<String>,
-        #[ts(type = "object")]
+        #[ts(type = "any")]
```

Comment:
Is this related to history events?

Reply:
Its part of the other PR where this one was built upon.